cmd/kube-controller-manager/app/OWNERS

```diff
@@ -0,0 +1 @@
+reviewers:
```

Comment:
@mikedanese makes lots of sense. I've removed it. Please review the kube-controller-manager/OWNERS file!